GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -3252,6 +3255,46 @@ subroutine get_GEOS( date_time, force_dtstep, met_path, met_tag,             &
     G5DAS_defs(11,:)=[character(len=40):: 'QLML    ','inst','inst1_2d_lfo_Nx','diag','S']    
     G5DAS_defs(12,:)=[character(len=40):: 'SPEEDLML','inst','inst1_2d_lfo_Nx','diag','S']    
 
+
+    ! -----------------------------------------------------------------------
+    !
+    ! define coupled land/atm DAS file specs (i.e., use bkg.lfo_* files) 
+    !
+    ! same as G5DAS except for file tag (column 3)
+
+    G5BKG_defs = G5DAS_defs
+
+    ! character(40):
+    !                            1         2         3         4
+    !                   1234567890123456789012345678901234567890     
+
+    G5BKG_defs( 1,3) = 'bkg.tavg1_2d_lfo                        '
```

Comment:
The "bkg.lfo_*" Collection names here need to be consistent with whatever is finalized in https://github.com/GEOS-ESM/GEOSgcm_App/pull/707